quipucords/quipucords/settings.py

```diff
@@ -283,9 +283,12 @@ def app_secret_key_and_path():
 VERBOSE_FORMATTING = (
     "%(levelname)s %(asctime)s %(module)s %(process)d %(thread)d %(message)s"
 )
-
-LOG_DIRECTORY = Path(env.str("LOG_DIRECTORY", str(BASE_DIR)))
+LOG_DIRECTORY = Path(env.str("QPC_LOG_DIRECTORY", str(DEFAULT_DATA_DIR / "logs")))
+# for convenience, create the default log directory
+if LOG_DIRECTORY.parent == DEFAULT_DATA_DIR:
+    LOG_DIRECTORY.mkdir(exist_ok=True, parents=True)
```

Comment:
I understand the convenience of this, but altering the filesystem in the settings module feels icky to me. 🤔

Reply:
fair enough.

Reply:
this was handled in a separate PR